src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import 'variables';
+@import 'stars';
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.products {
+  display: flex;
+}
+
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: $roboto-font;
+  color: $text-color;
+  font-weight: 400;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 16px;
+  }
+
+  &__name {
+    margin-top: 40px;
+  }
+
+  &__headline {
+    font-size: $headline-size;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0;
+  }
+
+  &__product-code {
+    margin: 0;
+    font-size: $subscript-size;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-top: 4px;
+  }
+
+  &__review-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__review-text {
+    font-size: $subscript-size;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: $headline-size;
+    line-height: 18px;
+    color: $secondary-color;
+    margin: 0;
+  }
+
+  &__price {
+    font-size: $price-text;
+    line-height: 18px;
+    font-weight: 700;
+    margin: 0;
+  }
+
+  &__image {
+    width: 100%;
+    height: 134px;
+    padding: 0;
+    margin-top: 16px;
+    position: relative;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    height: 100%;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 38px;
+    background-color: $accent-color;
+    font-size: $text-size;
+    font-weight: 700;
+    border: 1px solid $accent-color;
+    border-radius: 5px;
```

Comment:
Consider using a variable for the border color. It's a good practice to use variables for colors to keep consistency across your project.